modules/check_in/spec/sidekiq/travel_claim_base_job_spec.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CheckIn::TravelClaimBaseJob do
+  let(:mobile_phone) { '202-555-0123' }
+  let(:appointment_date) { '2023-05-15' }
+  let(:claim_number) { '1234' }
+  let(:facility_type) { 'oh' }
+  let(:template_id) { 'template-id-123' }
+  let(:job_opts) do
+    {
+      mobile_phone:,
+      appointment_date:,
+      template_id:,
+      claim_number:,
+      facility_type:
+    }
+  end
+
+  describe '#send_notification' do
+    subject { described_class.new.send_notification(job_opts) }
+
+    before do
+      allow_any_instance_of(VaNotify::Service).to receive(:send_sms)
+      allow_any_instance_of(described_class).to receive(:logger).and_return(Logger.new(nil))
+    end
+
+    it 'logs the notification and sends SMS' do
+      expect_any_instance_of(described_class).to receive(:log_sending_travel_claim_notification).with(job_opts)
+      expect_any_instance_of(described_class).to receive(:va_notify_send_sms).with(job_opts)
+      subject
+    end
+
+    context 'when an error occurs' do
+      before do
+        allow_any_instance_of(described_class).to receive(:va_notify_send_sms)
+          .and_raise(StandardError.new('Test error'))
+      end
+
+      it 'logs the failure and raises the error' do
+        expect_any_instance_of(described_class).to receive(:log_send_sms_failure)
```

Comment:
I think this should be testing the call to Logger and not the call the function within the class under test.